.rustfmt.toml

```diff
@@ -1,24 +1,23 @@
-# Basic
+# Non-default formatting configuration options: use with `cargo +nightly fmt  --all`
+binop_separator = "Back"
+chain_width = 80
+combine_control_expr = false
+comment_width = 100
+condense_wildcard_suffixes = true
 edition = "2021"
+format_code_in_doc_comments = true
+format_strings = true
+group_imports = "StdExternalCrate"
 hard_tabs = true
-max_width = 100
-use_small_heuristics = "Max"
-# Imports
 imports_granularity = "Crate"
-reorder_imports = true
-# Consistency
-newline_style = "Unix"
-# Misc
-chain_width = 80
-spaces_around_ranges = false
-binop_separator = "Back"
-reorder_impl_items = false
-match_arm_leading_pipes = "Preserve"
 match_arm_blocks = false
 match_block_trailing_comma = true
-trailing_comma = "Vertical"
+newline_style = "Unix"
+normalize_comments = true
+reorder_impl_items = true
 trailing_semicolon = false
+unstable_features = true
 use_field_init_shorthand = true
-# Format comments
-comment_width = 100
+use_small_heuristics = "Max"
```

Comment:
Maybe adding a comment like:

`# Applies `max_width` value or its default (`100`) if not specified.`

Reply:
Done